apipie-rails.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", ">= 4.1"
+  s.add_dependency "railties", ">= 4.1"
```

Comment:
is this a good version? I saw railtie is on 5 now.

Reply:
I'm just keeping whatever version is spec'ed here. Our gemfile will specify a more recent version, which will be honored given this is `>= 4.1`.

Reply:
👍